frontend/src/stores.js

```diff
@@ -5,3 +5,4 @@ import { writable } from "svelte/store";
  */
 export const selected = writable(undefined);
 export const selectedResource = writable(undefined);
+export const backendUrl = "";
```

Comment:
I recommend adding a comment that explains that when this string is empty, the back end server is the same one the GUI is served from, and that modifying this enables using a back end server running on a different port.